content-src/components/DiscoveryStreamBase/DiscoveryStreamBase.jsx

```diff
@@ -26,7 +26,7 @@ const ALLOWED_CSS_URL_PREFIXES = [
   "https://img-getpocket.cdn.mozilla.net/",
 ];
 const DUMMY_CSS_SELECTOR = "DUMMY#CSS.SELECTOR";
-let rickRollCache = []; // Cache of random probability values for a spoc position
+let rollCache = []; // Cache of random probability values for a spoc position
```

Comment:
Just changing this little bit of whimsical naming, mostly because now it's exposed as an actual option value in function call.